README.md

```diff
@@ -1,5 +1,8 @@
 # workflow
 
+[![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](https://www.apache.org/licenses/LICENSE-2.0)
+[![Maven Central](https://img.shields.io/maven-central/v/com.squareup.workflow/workflow-core.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.squareup.workflow%22)
```

Comment:
Is there any similar treatment we can do for Cocoapods? If not a badge, an id / version number?

cc @davidapgar @segiddins

Reply:
I think for Cocoapods it would be:
[![CocoaPods compatible](https://img.shields.io/cocoapods/v/Workflow.svg)]

Reply:
Sweet! Adding…

Reply:
Done.